client/src/templates/Introduction/components/Challenges.js

```diff
@@ -55,10 +57,45 @@ export class Challenges extends Component {
   }
 
   render() {
-    const { challengesWithCompleted, isProjectBlock } = this.props;
+    const { challengesWithCompleted, isProjectBlock, superBlock } = this.props;
+    const isGridMap = superBlock === SuperBlocks.RespWebDesignNew || superBlock;
```

Comment:
Since `superBlock` is always defined, we can drop `|| superBlock`. That'll stop the grid being enabled for all superblocks.